bin/naga.rs

```diff
@@ -77,6 +77,10 @@ fn main() {
                         panic!("Unknown profile: {}", string)
                     };
                 }
+                #[cfg(feature = "msl-out")]
+                "sizes-buffer-slot" => {
```

Comment:
I don't think this is useful right now. I.e. there aren't any command line arguments for the resource binding map.